R/constants.R

```diff
@@ -19,6 +19,7 @@ FLAG_PROTECT_FLOAT <- "FLAG_PROTECT_FLOAT"
 # Time constants
 HISTORICAL_YEARS <- 1971:2010
 FUTURE_YEARS <- seq( 2015, 2100, 5 )
+IMF_FUTURE_YEARS <- seq( 2010, 2020, 1 )
```

Comment:
Yes, but it only allows certain ones (specifically the ones I've added above).  The check mark and cross-mark would be useful to have as options.